test/onnx/onnx_test.cpp

```diff
@@ -4311,6 +4311,26 @@ TEST_CASE(reducel1_test)
     EXPECT(p == prog);
 }
 
+TEST_CASE(reducel1_dyn_test)
+{
+    migraphx::program p;
+    auto* mm = p.get_main_module();
+    // a shape with 4 dynamic dimensions
+    auto l0 = mm->add_parameter(
+        "x",
+        migraphx::shape{migraphx::shape::float_type, {{3, 3, 0}, {3, 5, 0}, {4, 6, 5}, {5, 7, 6}}});
+    auto abs_l0 = mm->add_instruction(migraphx::make_op("abs"), l0);
+    auto sum_l0 = mm->add_instruction(migraphx::make_op("reduce_sum", {{"axes", {-2}}}), abs_l0);
+    auto sq_l0  = mm->add_instruction(migraphx::make_op("squeeze", {{"axes", {-2}}}), sum_l0);
+    mm->add_return({sq_l0});
+
+    migraphx::onnx_options options;
+    options.map_dyn_input_dims["x"] = {{3, 3}, {3, 5}, {4, 6, 5}, {5, 7, 6}};
+    auto prog                       = migraphx::parse_onnx("reducel1_dyn_test.onnx", options);
+
+    EXPECT(p == prog);
+}
+
```

Comment:
Why is there an `abs` instruction?

Reply:
I don't know.  The instructions were copied from the previous test, `reducel1_test`

Reply:
Removed the abs--wait to see if test result is the same

Reply:
Abs turns out to be a necessary part of L1_reduce.  Restored.